frontend/class-twitter.php

```diff
@@ -158,6 +158,9 @@ protected function description() {
 		if ( is_singular() ) {
 			$meta_desc = $this->single_description();
 		}
+		elseif ( is_home() && ! is_front_page() ) {
```

Comment:
There is `is_posts_page()` in Frontend class, but it's actually different — it tells if it's home regardless if it's a _static page_ "kind" of a home. It also has `is_home_static_page()` for that, but neither is static though.

Move implementation for the whole bunch to Utils?